backend/business-partner-agent/src/main/java/org/hyperledger/bpa/impl/aries/ConnectionManager.java

```diff
@@ -83,6 +89,27 @@
     @Inject
     ObjectMapper mapper;
 
+    /**
+     * Create a connection based on a public did that is registered on a ledger.
+     * 
+     * @param alias optional connection alias
+     */
+    public Optional<CreateInvitationResponse> createConnectionInvitation(@NonNull String alias) {
+        Optional<CreateInvitationResponse> result = Optional.empty();
+        try {
+            result = ac.connectionsCreateInvitation(
+                    CreateInvitationRequest.builder()
+                            .serviceEndpoint(acapyEndpoint)
```

Comment:
Endpoint shouldn't be needed

Reply:
You are correct, i have removed it, though i'm not sure how that it is the case, see update version.